lib/matplotlib/font_manager.py

```diff
@@ -263,10 +264,9 @@ def findSystemFonts(fontpaths=None, fontext='ttf'):
             fontfiles.update(win32InstalledFonts(fontext=fontext))
         else:
             fontpaths = X11FontDirectories
-            fontfiles.update(get_fontconfig_fonts(fontext))
-            # check for OS X & load its fonts if present
             if sys.platform == 'darwin':
-                fontfiles.update(OSXInstalledFonts(fontext=fontext))
+                fontpaths = [*X11FontDirectories, *OSXFontDirectories]
```

Comment:
The new list expansions are nice, but this is just `fontpaths += OSXFontDirectories`, no?